pg_replicate/Cargo.toml

```diff
@@ -60,6 +65,7 @@ tracing-subscriber = { workspace = true, default-features = true, features = [
 [features]
 bigquery = ["dep:gcp-bigquery-client", "dep:prost"]
 duckdb = ["dep:duckdb"]
+delta = []
```

Comment:
The `delta` feature should enable the `deltalake` dependency, no?

Reply:
yes. let me fix